libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -48,9 +48,37 @@ void copyMissingTags(StructurallyDocumentedAnnotation& _target, set<CallableDecl
 	for (auto const& iterator: _target.docTags)
 		existingTags.insert(iterator.first);
 
-	for (auto const& [tag, content]: sourceDoc.docTags)
-		if (tag != "inheritdoc" && !existingTags.count(tag))
-			_target.docTags.emplace(tag, content);
+	for (auto it = sourceDoc.docTags.begin(); it != sourceDoc.docTags.end();)
```

Comment:
Please either add a comment to explain what this does or extract it into a function which is then self-explanatory through its name.

Reply:
it's used to get information about the return parameters and their names so they can be adjusted when copied in case of functions.